README.md

```diff
@@ -35,19 +35,24 @@ striking similarity to JavaScript has provoked criticism, the accusation of
 having too relaxed design principles for transferring money securely being
 thrown around on the Internet frequently. And indeed, Solidity features odd
 design choices. To name a few: (1) It doesn't have under- and overflow
-protection, (2) interfaces are not enforced and (3) similar modifiers are
-easily confused as their naming is virtually indistinguishable.
+protection, (2) interfaces are not enforced, (3) similar modifiers are easily
+confused as their naming is virtually indistinguishable, (4) code reuse is
+implemented through multiple inheritance, a pattern known for it's complexity
+and corner cases and often avoided even by seasoned programmers, that can lead
+to some non trivial bugs [13] and (5) inline assembly is hard to write and,
+even more important, difficult to read, making the smart contract difficult to
```

Comment:
Isn't it "more importantly"? Not sure.

Reply:
I can remove it and simplify the sentence

Reply:
ok